crypto/bn/bn_ctx.c

```diff
@@ -86,8 +86,11 @@ struct bignum_ctx {
     int too_many;
     /* Flags. */
     int flags;
+    /* The library context */
+    OPENSSL_CTX *libctx;
 };
 
+#ifndef FIPS_MODE
```

Comment:
Where is the #endif associated with this #ifndef ?

Reply:
I try to do that too for longer spans, but forgot in this instance. Added now.